bootstrap/pkg/apis/apps/group.go

```diff
@@ -47,6 +47,8 @@ const (
 	KustomizationFile      = "kustomization.yaml"
 	KustomizationParamFile = "params.env"
 	DefaultCacheDir        = ".cache"
+	KubeflowRepoName       = "kubeflow"
+	ManifestsRepoName      = "manifests"
```

Comment:
KubeflowRepoName, and ManifestsRepoName are used as the name field in Spec.Repos. So they don't necessarily need to match the actual name on github. I didn't want to break existing code which was using KubeflowRepo and ManifestsRepo as github names.

We can dedupe in the future because the old values should eventually not be needed.